cmd/main.go

```diff
@@ -1,14 +1,75 @@
 package main
 
 import (
+	"errors"
 	"fmt"
+	"github.com/hashgraph/hedera-sdk-go"
+	hederaClient "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera"
 	"github.com/limechain/hedera-eth-bridge-validator/app/persistence"
+	"github.com/limechain/hedera-eth-bridge-validator/app/persistence/status"
+	consensusmessage "github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/consensus-message"
+	cryptotransfer "github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/crypto-transfer"
 	"github.com/limechain/hedera-eth-bridge-validator/config"
 	"github.com/limechain/hedera-watcher-sdk/server"
+	log "github.com/sirupsen/logrus"
+	"os"
 )
 
 func main() {
+	initLogger()
 	configuration := config.LoadConfig()
-	persistence.RunDb(configuration.Hedera.Validator.Db)
-	server.NewServer().Run(fmt.Sprintf(":%s", configuration.Hedera.Validator.Port))
+	hederaClient := hederaClient.NewHederaClient(configuration.Hedera.MirrorNode.ApiAddress, configuration.Hedera.MirrorNode.ClientAddress)
+	server := server.NewServer()
+
+	db := persistence.RunDb(configuration.Hedera.Validator.Db)
```

Comment:
When we run the db, we must `defer` the db connection

Reply:
No need to defer the db connection, the driver handles it itself.